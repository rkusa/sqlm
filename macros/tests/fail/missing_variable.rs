fn main() {
    sqlm_macros::sql!("{id}");
}
