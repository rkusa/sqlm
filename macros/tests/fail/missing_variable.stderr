error[E0425]: cannot find value `id` in this scope
 --> tests/fail/missing_variable.rs:2:5
  |
2 |     sqlm_macros::sql!("{id}");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
  |
  = note: this error originates in the macro `sqlm_macros::sql` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider importing this function
  |
1 + use std::process::id;
  |
